RELEASE_NOTES.md

```diff
@@ -68,6 +71,12 @@
 - Fixed a bug where the `--filter` option to `podman ps` did not properly handle filtering by healthcheck status ([#11687](https://github.com/containers/podman/issues/11687)).
 - Fixed a bug where the `podman run` and `podman start --attach` commands could race when retrieving the exit code of a container that had already been removed resulting in an error (e.g. by an external `podman rm -f`) ([#11633](https://github.com/containers/podman/issues/11633)).
 - Fixed a bug where the `podman generate kube` command would add default environment variables to generated YAML.
+- Fixed a bug where the `podman generate kube` command would add the default CMD from the image to generated YAML ([#11672](https://github.com/containers/podman/issues/11672)).
+- Fixed a bug where the `podman rm --storage` command could fail to remove containers under some circumstances ([#11207](https://github.com/containers/podman/issues/11207)).
+- Fixed a bug where the CNI plugins were unable to locate state information when Podman was run as root, leading to a failure to properly tear down container networking ([#10745](https://github.com/containers/podman/issues/10745) and [#10806](https://github.com/containers/podman/issues/10806)).
```

Comment:
Please remove this, it is not fixed in 3.4

Reply:
Oh, I thought that the revert also fixed this by unsetting XDG_RUNTIME_DIR for the CNI plugins. Removing, then...

Reply:
Yes but only for 4.0, the 3.4 code does not use the network interface